ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -104,65 +128,124 @@ Rectangle {
         }
 
         StatusListView {
+            id: walletAccountsListView
+
             objectName: "walletAccountsListView"
             spacing: Style.current.smallPadding
             Layout.fillWidth: true
             Layout.fillHeight: true
             Layout.topMargin: Style.current.halfPadding
+            Layout.leftMargin: -Style.current.padding
+            Layout.rightMargin: -Style.current.padding
 
             // ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
 
+            readonly property Item firstItem: count > 0 ? itemAtIndex(0) : null
+
             delegate: StatusListItem {
                 objectName: "walletAccountItem"
-                width: ListView.view.width
-                highlighted: RootStore.currentAccount.name === model.name
+                width: ListView.view.width - Style.current.padding * 2
+                highlighted: !ListView.view.footerItem.button.highlighted &&
+                             RootStore.currentAccount.name === model.name
+                anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
this looks a bit complex. ListView has negative margins, hence it stretches outside its parent. The delegate has the ListView's width minus the negative margins and it's horizontally centered to it parent. I'm sure we can do something better here